sys/dev/athn/if_athn_usb.c

```diff
@@ -49,8 +49,9 @@
 #include <net/if_var.h>
 // #include <net/if_types.h>
 
+#include <net/if_arp.h>
 #include <netinet/in.h>
-#include <netinet/if_ether.h>
+//#include <netinet/if_ether.h>
```

Comment:
Can we remove this if we don't need it?

Reply:
If you look in the file in question, we have for instance these:
// #include <dev/ic/athnreg.h>
// #include <dev/ic/ar5008reg.h>
// #include <dev/ic/athnvar.h>

So to be consistent these should be removed as well.

Reply:
Yes, it will be generally great if someone can have a look at the first patch in our mobica_dev_14 branch - the one from Sylwester (sygl) - and check which original FreeBSD source files were changed. In the future, we'll have to force these changes to upstream to FreeBSD repo...

Reply:
Ok, sure. Let's do this in the cleanup phase.